pkg/guest/guest.go

```diff
@@ -97,6 +97,11 @@ func (d *Default) getGuestCmd(envs map[string]string, cluster *v2.Cluster, mount
 			envs = maps.MergeMap(baseEnvs, i.Env)
 		}
 		mapping := expansion.MappingFuncFor(envs)
+
+		for _, cmd := range i.EntryPoint {
+			command = append(command, expansion.Expand(cmd, mapping))
+		}
+
 		if len(i.Cmd) != 0 {
```

Comment:
Is the judgement here unnecessary?

Reply:
I think the judgetment here is unnecessary, bacause range a nil slice is valid.

Reply:
You can delete this if Judgement